src/datum_conf.c

```diff
@@ -177,6 +177,8 @@ const T_DATUM_CONFIG_ITEM datum_config_options[] = {
 		.required = false, .ptr = &datum_config.datum_pooled_mining_only, 	.default_bool = true },
 	{ .var_type = DATUM_CONF_INT, 		.category = "datum", 		.name = "protocol_global_timeout",		.description = "If no valid messages are received from the DATUM server in this many seconds, give up and try to reconnect",
 		.required = false, .ptr = &datum_config.datum_protocol_global_timeout, 	.default_int = 60 },
+	{ .var_type = DATUM_CONF_BOOL, 		.category = "datum", 		.name = "proxy_protocol",		.description = "Enable support for the PROXY protocol",
```

Comment:
This would be a stratum category, not datum

Maybe

```suggestion
	{ .var_type = DATUM_CONF_BOOL, 		.category = "stratum", 		.name = "trust_proxy",		.description = "Enable trusting remote IPs forwarded using the PROXY protocol",
```